docs/automated-tests/troubleshooting/element-not-found.md

```diff
@@ -1,32 +1,32 @@
 ---
 title: How to Troubleshoot Element Not Found 
 page_title: How to Troubleshoot Element Not Found?
-description: "Test Studio - debug element not found error. How to understand what causes the test to fail. What options I have to debug or troubleshoot the element not found failing test. My test fails with Element not found exception. My test fails to locate an element on page." 
+description: "Learn how to debug the element not found error in Test Studio. How to understand what causes the test to fail and what troubleshoot approach to take when your test fails to locate an element on page." 
 position: 1
 ---
 # Troubleshoot 'Element Not Found' Error
 
 A common failure when executing Test Studio tests is that a particular element is not found on the page during the test run.
 
-In some cases the element is visibly available on the page, but is still reported as not found by Test Studio. This tutorial  will guide you through the troubleshooting workflow for this type of a failure and what can be causing it.
+In some cases, the element is visibly available on the page but still reported as not found by Test Studio. This tutorial will guide you through the troubleshooting workflow for this type of failure and what can be causing it.
 
 ## Record the Test Scenario
 
-For this tutorial we use a sample login application. The task is to fill the login credentials and click on the _Log In_ button, after logging in we get a greeting _Welcome, Test Studio_!
+In this tutorial, we will use a sample login application. The task is to fill in the login credentials and click on the _Log In_ button. After logging in, we get a _Welcome, Test Studio_ greeting.
 
 ![sample app](/img/automated-tests/troubleshooting/element-not-found/1SampleApp.png)
 
 We’ve already launched <a href="/automated-tests/recording/overview#start-a-recording-session" target="_blank">Test Studio’s recorder and recorded</a> the following test steps:
```

Comment:
- Is it expected from the reader to perform these steps? If it is, be specific and provide the required instructions.
- I believe that this is much more a sample scenario and not a tutorial. In a tutorial, the readers perform the steps by following the instructions. Here you are explaining what happens in the example, but you don't expect the user to follow instructions. I suggest to remove the word "tutorial" from the article's intro.